vars/spinInfra.groovy

```diff
@@ -7,21 +7,19 @@ import uk.gov.hmcts.contino.MetricsPublisher
 import uk.gov.hmcts.pipeline.AKSSubscriptions
 import uk.gov.hmcts.contino.RepositoryUrl
 
-def call(productName, environment, tfPlanOnly, subscription) {
-  call(productName, null, environment, tfPlanOnly, subscription)
-}
-
-def call(product, component, environment, tfPlanOnly, subscription) {
-  call(product, component, environment, tfPlanOnly, subscription, "")
-}
-
-def call(product, component, environment, tfPlanOnly, subscription, deploymentTarget) {
```

Comment:
🚫 All existing method signatures must be kept or all existing consumers updated. You'll need to restore the functions and then convert them to the new syntax in a call method as these were doing for other method compatibility before.

https://github.com/search?q=org%3Ahmcts%20spinInfra%20NOT%20repo%3Ahmcts%2Fcnp-jenkins-library&type=code

Likely many of the repositories are dead but we're currently stuck with them even though they really shouldn't be using `spinInfra`